spring-rabbit/src/test/java/org/springframework/amqp/rabbit/core/RabbitAdminIntegrationTests.java

```diff
@@ -336,6 +346,52 @@ public void testQueueDeclareBad() {
 		this.rabbitAdmin.deleteQueue(queue.getName());
 	}
 
+	@Test
+	public void testDeclareDelayedExchange() throws Exception {
+		DirectExchange exchange = new DirectExchange("test.delayed.exchange");
+		exchange.setDelayed(true);
+		Queue queue = new Queue(UUID.randomUUID().toString(), true, false, false);
+		Binding binding = new Binding(queue.getName(), DestinationType.QUEUE, exchange.getName(), queue.getName(), null);
+
+		try {
+			this.rabbitAdmin.declareExchange(exchange);
+		}
+		catch (AmqpIOException e) {
+			if (RabbitUtils.isExchangeDeclarationFailure(e)
+					&& e.getCause().getCause().getMessage().contains("invalid exchange type 'x-delayed-message'")) {
```

Comment:
This doesn't path for me:
```
connection error; protocol method: #method<connection.close>(reply-code=503, reply-text=COMMAND_INVALID - unknown exchange type 'x-delayed-message', class-id=40, method-id=10)
```
My RabbitMQ version is RabbitMQ 3.5.4, Erlang 18.0.
Or that is the Windows RabbitMQ variant...